### PR TITLE
refactoring query issue for adding different status

### PR DESCRIPTION
**Pull request by dhiemaz**
@ankitpokhrel, I want to contribute as part of hacktoberfest to issue #488 

for single status
```
$jira issue list --assignee prawira.dimas.yudha@gmail.com --status "TO DO"
```

for multiple status
```
$issue list --assignee prawira.dimas.yudha@gmail.com --status "TO DO","BLOCKED"
```

### CODE REVIEW

**Comment by ankitpokhrel** [comment]
Hi @dhiemaz, thank you for the PR. While this works, let's use `stringArray` for consistency reasons. We do something [similar for `labels`](https://github.com/ankitpokhrel/jira-cli/blob/c3fe62128748ed2a9a95bb353e805fc71ff49393/internal/cmd/issue/list/list.go#L158).

**Comment by dhiemaz** [comment]
Hi @ankitpokhrel thank you for your feedback, ok let me change it.

File: internal/query/issue.go [hunk 1]
```diff
@@ -62,19 +62,26 @@ func (i *Issue) Get() string {
...
 		if len(statuses) > 1 {
+			q.FilterBy("type", i.params.IssueType).
+				FilterBy("resolution", i.params.Resolution).
+				FilterBy("priority", i.params.Priority).
+				FilterBy("reporter", i.params.Reporter).
+				FilterBy("assignee", i.params.Assignee).
+				FilterBy("component", i.params.Component).
+				FilterBy("parent", i.params.Parent)
+
 			q.In("status", statuses...)
```
Comment (chripo) [inline internal/query/issue.go:75]: Is it not enough to append the `statuses` filter
```go
q.FilterBy("type" ...
statuses := strings.Split(i.params.Status, ",")
if len(statuses) > 1 {
   q.In("status", statuses)
}
```
or even without `if`

**Comment by ankitpokhrel** [comment]
Closing in favor of #573. Thank you for your contribution!

**Pull request closed without merging**